gobblin-utility/src/main/java/org/apache/gobblin/util/logs/LogCopier.java

```diff
@@ -85,10 +86,7 @@
  *           .useDestFileSystem(FileSystem.get(URI.create(destFsUri), new Configuration()))
  *           .readFrom(new Path(srcLogDir))
  *           .writeTo(new Path(destLogDir))
- *           .useInitialDelay(60)
```

Comment:
I found several initial-delay related comments in the class, please clean as well.

Reply:
Thanks for pointing it out. Will clean it up.